python/pyiceberg/expressions/visitors.py

```diff
@@ -185,7 +185,7 @@ def _(obj: Or, visitor: BooleanExpressionVisitor[T]) -> T:
     return visitor.visit_or(left_result=left_result, right_result=right_result)
 
 
-def bind(schema: Schema, expression: BooleanExpression, case_sensitive: bool) -> BooleanExpression:
+def bind(schema: Schema, expression: BooleanExpression, case_sensitive: bool = True) -> BooleanExpression:
```

Comment:
I also made this one mandatory again, so we don't accidentally forget to set it ..